packages/react-sdk/src/component-generator.ts

```diff
@@ -246,6 +255,10 @@ export const generateJsxElement = ({
     generatedElement += children;
     generatedElement += `</Fragment>\n`;
     generatedElement += `)}\n`;
+  } else if (instance.component === blockComponent) {
+    if (instance.children.length > 1) {
+      generatedElement += children;
+    }
```

Comment:
no need to check instance.children. block template is managed already in the other place

Reply:
Wow really, thanks